lib/pages/cases.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:flutter/material.dart';
+
+class Cases extends StatefulWidget {
+  @override
+  _CasesState createState() => _CasesState();
+}
+
+class _CasesState extends State<Cases> {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Text('Cases'),
+        centerTitle: true,
+      ),
+      body: Center(
+        child: Column(
+          mainAxisAlignment: MainAxisAlignment.center,
+          children: <Widget>[
+            Text(
+              'Cases',
+            ),
```

Comment:
Add this style   
`style: TextStyle(
                  color: Colors.red[300],
                  fontWeight: FontWeight.bold,
                  fontSize: 20.0,),
                  textAlign: TextAlign.center,
            )`